python2_nbd_client.py

```diff
@@ -30,17 +31,21 @@ class new_nbd_client(object):
     FLAG_SEND_FLUSH = (1 << 2)
 
     NBD_OPT_EXPORT_NAME = 1
+    NBD_OPT_STARTTLS = 5
+    NBD_REP_ACK = (1)
+    cflags = (1 << 0)
```

Comment:
Why `(1 << 0)` rather than `1`?

Reply:
That was just how it was defined in the examples I looked at: 
`#define NBD_FLAG_FIXED_NEWSTYLE (1 << 0) `
(https://github.com/NetworkBlockDevice/nbd/blob/master/cliserv.h)
I assumed thats how it had to be done seeing as the other flags are defined in this format

Reply:
I think that's just for readability: consistency of appearance next to the other flags that have a number other than 0.

Anyway, I think we should either have a comment saying that this client_flags bitmap contains just the NBD_FLAG_C_FIXED_NEWSTYLE flag, or else do it by means of variable-names:
`NBD_FLAG_C_FIXED_NEWSTYLE = (1 << 0)` (or just `1`) and then
`cflags = NBD_FLAG_C_FIXED_NEWSTYLE`